public/404.html

```diff
@@ -1,67 +1,95 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>The page you were looking for doesn't exist (404)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  .rails-default-error-page {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
+<html lang="en" class="govuk-template ">
+  <head>
+    <meta charset="utf-8">
+    <title>There is a problem with the service</title>
```

Comment:
Maybe for 404 it should be "Page not found" or "The page you were looking for doesn't exist (404)"

Reply:
Fixed